packages/memory/README.md

```
@@ -0,0 +1,250 @@
+
+// Get statistics
+const stats = await memory.getIndexStats();
+console.log(`Total memories: ${stats.total_memories}`);
+
+// Close when done
+memory.close();
+```
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)